docs/Configuring-Orion/Using-PostgreSQL.md

```diff
@@ -0,0 +1,59 @@
+description: Storing Information with PostgreSQL 
+<!--- END of page meta data -->
+
+# Store Information with PostgreSQL
+
+PostgreSQL is an object-relational database you can use to store private transaction payloads for Orion. Consider using this option as an aid in [disaster recovery](Disaster-Recovery.md).
+
+## Prerequisites
+
+* [PostgreSQL](https://www.postgresql.org/)
+
+## Set up your PostgreSQL database
+
+In your database:
+
+1. Create the `store` table by using the provided DDL script, as shown in the following example. Substute your own values for `HOST-NAME`, `USER-NAME`, and `DATABASE-NAME`.
```

Comment:
Even thought the script is now provided with the binary I think we're going to need to tell them where it is.  @jframe -> will in be in the location where the binary is installed?

Reply:
@MadelineMurray <installation_dir>/orion/database/postgres_ddl.sql